android-sdk/tasks/main.yml

```diff
@@ -156,6 +156,16 @@
     - install-sdk
 
 -
+  name: "Prompt to accept license"
+  pause:
+    prompt: |
+      Press enter to accept Android's SDK license and install the SDK
+      https://developer.android.com/studio/index.html#linux-bundle
+  when:
```

Comment:
The `when` condition can be removed since  the pause module ignores it.

Do you think it makes sense do add some text to just press enter and ignore it if running for a second time?

Reply:
ignore my previous message :)

For some reason it never validates to "true" in my minishift vm.

Reply:
Actually the way it checks for the android sdk path is not working, it should do a folder check with the "oc rsh" command.